python/trezorlib/tests/device_tests/test_msg_resetdevice_shamir.py

```diff
@@ -158,11 +158,14 @@ def input_flow():
 def validate_mnemonics(mnemonics, threshold, expected_secret):
```

Comment:
Clarify the name expected_secret. Something like expected_encrypted_master_secret or expected_ems.

Reply:
Okay. But this a) test, b) needs refactor anyway. I think it makes more sence to fix it later.

Reply:
Done.